### DISCUSSION TITLE
extract world space vertices of an animated model?

### DISCUSSION THREAD

**Discussion by ThomasCartier** [bevyengine/bevy#11046] [Q&A] [answered]
Hi,

What would be the best approach to extract the world space vertices of an animated model at every step? It's from a humanoid.

Reading the doc, I may have missed the info.
Thanks,

**Comment by nicopap** [comment, marked as answer]
The animated vertices coordinates are computed on the GPU in a vertex shader. You would use the `vertex` function in `mesh.wgsl` here <https://github.com/bevyengine/bevy/blob/ced216f59a11f70d4e1822d267486ea9f75759e7/crates/bevy_pbr/src/render/mesh.wgsl#L32>

Depending on your use case this might not be useful.

**Reply by ThomasCartier** [reply]
Thanks!
